the arm builders have nonfunctional nft, so the command fails
just hardcoded it for now.
--
diff --git a/setup.py b/setup.py
index 97461ce..d25d664 100644
--- a/setup.py
+++ b/setup.py
@@ -276,11 +276,7 @@ for e in ['ip6tables', 'iptables-restore', 'ip6tables-restore']:
         print("ERROR: could not find required binary '%s'" % (e), file=sys.stderr)
         sys.exit(1)
 
-(rc, out) = cmd([iptables_exe, '-V'])
-if rc != 0:
-    raise OSError(errno.ENOENT, "Could not find version for '%s'" % \
-                  (iptables_exe))
-version = re.sub('^v', '', re.split('\s', str(out))[1])
+version = '1.8.9'
 print("Found '%s' version '%s'" % (iptables_exe, version))
 if version < "1.4":
     print("WARN: version '%s' has limited IPv6 support. See README for details." % (version), file=sys.stderr)
